apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html

```diff
@@ -63,9 +63,14 @@ <h5>
                             {{localeData?.advanced_receipts}}
                             <div>
                                 <span
-                                    class="number-total">{{receiptsSummaryData?.advance?.symbol}}{{receiptsSummaryData?.advance?.amount
+                                    class="number-total d-inline-flex">
+                                    <!-- {{receiptsSummaryData?.advance?.symbol}}{{receiptsSummaryData?.advance?.amount
                                     |
-                                    giddhCurrency}}/-</span>
+                                    giddhCurrency}}/- -->
+
+                                    <amount-field [amount]="receiptsSummaryData?.advance?.amount" [currencySymbol]="receiptsSummaryData?.advance?.symbol" [currencyCode]="false">
```

Comment:
Do check currencyCode is not passed but currencySymbol is provided which will not make it RTL for Gulf countries.

Reply:
added